test/data/testSP.py

```diff
@@ -14,8 +14,8 @@
 
 import numpy
 
-from testall import skipForParser
-from testall import skipForLogfile
+from skip import skipForParser
+from skip import skipForLogfile

```

Comment:
These are giving me an `ImportError`, unless I change them to relative imports. Same with `testSPun` and `testScan`.